src/data/vault.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Vault",
```

Comment:
Added this since we seem to have settled on `data` as our one home for product data. Def happy to wait for upstream changes when #67 merges.